### ISSUE TITLE
Link to Published Source Code

### ISSUE THREAD

**Issue by jjorissen52** [ArtemGr/Sidekick#229]
Hello, this is a very cool bot, and we enjoy using it on our Discord. That said, this project is published under the MIT license and appears on multiple lists as "Open Source" software. However, the source is not published here, or, as far as I can tell, anywhere else. 

Having the documentation for the bot published on GitHub as "Open Source" under the MIT license is misleading to anyone who isn't paying close attention, to say the very least. To back this up, I have observed that at the time of this writing, 26 people have forked this repo, and those people either forked essentially an empty repository on purpose, or they thought this repo contained source code. 

So my request is this:

Make it very clear in the description that this repository does not contain "Dice and LFG bot for Discord," but that it instead contains some documentation for the above, and, if it actually is open source, please provide a link somewhere in the README to where that source code is hosted.

**Comment by CommanderTso** [comment]
Agreed - this is a great bot, but this is not in fact currently an open source project, and it should either be labeled correctly or the source code should be available.  Doesn't take away from the work that has gone in - the bot is awesome.

**Comment by cromlyngames** [comment]
if it was actually open soruce, we'd at least be able to contribute to resolving some of the open issues.

**Comment by nutjob4life** [comment]
All of the above. Respect for the work that's gone into the bot and kudos on making money on the Patreon, but this is a *closed-source* project that's essentially **abusing** GitHub in order to give it an issue tracker.
